### PR TITLE
Add what's new in January

### PR DESCRIPTION
**Pull request by BillWagner**
Updated and new articles in January.

[internal review link](https://review.docs.microsoft.com/en-us/dotnet/whats-new/?branch=pr-en-us-16931) to the updated index file.

### CODE REVIEW

File: docs/whats-new/dotnet-2020-01.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+# .NET docs: What's new for January 2020
+
+Welcome to what's new in .NET docs for January 2020. This article lists some of the major changes to docs during this period.
+
+## Architecture guides
+
+### New articles
+
+- [Caching in a cloud-native app](../architecture/cloud-native/azure-caching.md) - Chapter 5 octupdates
```
Comment (mairaw) [inline docs/whats-new/dotnet-2020-01.md]: I know this was the PR name but updates will read better

```suggestion
- [Caching in a cloud-native app](../architecture/cloud-native/azure-caching.md) - Chapter 5 updates
```

File: docs/whats-new/dotnet-2020-01.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+- [Database-per-microservice](../architecture/cloud-native/database-per-microservice.md) - Chapter 5 octupdates
```
Comment (mairaw) [inline docs/whats-new/dotnet-2020-01.md]:
```suggestion
- [Database-per-microservice](../architecture/cloud-native/database-per-microservice.md) - Chapter 5 updates
```

File: docs/whats-new/dotnet-2020-01.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+- [Elasticsearch in a cloud-native app](../architecture/cloud-native/elastic-search-in-azure.md) - Chapter 5 octupdates
```
Comment (mairaw) [inline docs/whats-new/dotnet-2020-01.md]:
```suggestion
- [Elasticsearch in a cloud-native app](../architecture/cloud-native/elastic-search-in-azure.md) - Chapter 5 updates
```

File: docs/whats-new/dotnet-2020-01.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+- [Boxing and Unboxing (C# Programming Guide)](../csharp/programming-guide/types/boxing-and-unboxing.md) - Clarify CLR acronym
+- [\<summary> (C# programming guide)](../csharp/programming-guide/xmldoc/summary.md) - Update XML documentation comments
+
+## .NET Framework
+
+### Updated articles
+
+- [Runtime Directives (rd.xml) Configuration File Reference](../framework/net-native/runtime-directives-rd-xml-configuration-file-reference.md) - Fix formatting of rd.xml documentation
+- [Unsupported scenarios](../framework/wcf/feature-details/unsupported-scenarios.md) - Acrolinx 12/30
```
Comment (mairaw) [inline docs/whats-new/dotnet-2020-01.md]: I thought we were ignoring Acrolinx PRs? This one mostly changed trailing white spaces.

```suggestion
```

File: docs/whats-new/dotnet-2020-01.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+- [Web services protocols interoperability guide](../framework/wcf/feature-details/web-services-protocols-interoperability-guide.md) - Replace some fwlinks, style-guide improvements
+- [Firewall instructions](../framework/wcf/samples/firewall-instructions.md) - Acrolinx 12/30
```
Comment (mairaw) [inline docs/whats-new/dotnet-2020-01.md]: same here

```suggestion
```

File: docs/whats-new/dotnet-2020-01.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+- [WS-AtomicTransaction Configuration MMC snap-in](../framework/wcf/ws-atomictransaction-configuration-mmc-snap-in.md) - Acrolinx 12/30
```
Comment (mairaw) [inline docs/whats-new/dotnet-2020-01.md]: same here

```suggestion
```

File: docs/whats-new/toc.yml [hunk 7]
```diff
@@ -2,8 +2,9 @@
   href: index.yml
   expanded: true
   items:
+    - name: 2020 - January
```
Comment (mairaw) [inline docs/whats-new/toc.yml]: this is not consistent with the 2019 node.
Shouldn't this be a 2020 node with a January child?

**Review by mairaw — APPROVED** [review]
LGTM. Left a few suggestions for you to consider.

File: docs/whats-new/toc.yml (hunk 7 above)
Comment (mairaw) [reply docs/whats-new/toc.yml]: Ah, never mind. Just saw your comment. Seems like an eng bug that should be fixed.

File: docs/whats-new/dotnet-2020-01.md (hunk 4 above)
Comment (BillWagner) [reply docs/whats-new/dotnet-2020-01.md]: Hmm. I thought we had too. I must not have merged that change yet. Good catch.

File: docs/whats-new/toc.yml (hunk 7 above)
Comment (BillWagner) [reply docs/whats-new/toc.yml]: I agree. But it won't be deployed in time before Feb becomes the "what's new" page.

File: docs/whats-new/dotnet-2020-01.md [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+### New articles
+
+- [Working with Interop Exceptions in Unmanaged Code](../standard/native-interop/exceptions-interoperability.md) - Added Longjump interop limitations documentation.
+- [How to migrate from Newtonsoft.Json to System.Text.Json](../standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md) - New article - Migrate from Newtonsoft.Json
+
+### Updated articles
+
+- [Binary serialization](../standard/serialization/binary-serialization.md) - Binary serialization improvements
+- [How to serialize and deserialize (marshal and unmarshal) JSON in .NET](../standard/serialization/system-text-json-how-to.md) - New article - Migrate from Newtonsoft.Json
+- [How to migrate from Newtonsoft.Json to System.Text.Json](../standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md) - Add table of differences to Migrate from Newtonsoft.Json doc
```
Comment (pkulikov) [inline docs/whats-new/dotnet-2020-01.md]: That same article is listed in the "New articles" section.

**Pull request merged**